src/Symfony/Bridge/PhpUnit/DeprecationErrorHandler.php

```diff
@@ -116,9 +116,16 @@ public static function register($mode = 0)
             }
 
             if (isset($trace[$i]['object']) || isset($trace[$i]['class'])) {
-                $class = isset($trace[$i]['object']) ? get_class($trace[$i]['object']) : $trace[$i]['class'];
-                $method = $trace[$i]['function'];
-                $Test = $UtilPrefix.'Test';
+                if (isset($trace[$i]['class']) && 'Symfony\Bridge\PhpUnit\SymfonyTestsListener' === $trace[$i]['class']) {
```

Comment:
Do we have to deal with the fact that this might be the legacy version?
 ie. \Symfony\Bridge\PhpUnit\Legacy\SymfonyTestsListener

Reply:
That's no problem as the trait is called by the listener. That's why we see the listener here.

Reply:
@xabbuh and that's precisely what @alexpott is saying: we have 2 listeners. you deal with only one of them.

Reply:
hum indeed, fixed